enterprise/server-enterprise/src/main/java/org/neo4j/server/enterprise/EnterpriseServerSettings.java

```diff
@@ -29,29 +30,22 @@
 import static org.neo4j.kernel.configuration.Settings.setting;
 
 @Description("Settings available in the Enterprise server")
-public interface EnterpriseServerSettings
+public class EnterpriseServerSettings implements LoadableConfig
 {
-    @Description( "Configure the operating mode of the database -- 'SINGLE' for stand-alone operation, 'HA' for " +
-            "operating as a member in a cluster, 'ARBITER' for an HA-only cluster member with no database, " +
-            "CORE for a core member of a Causal Clustering cluster, or READ_REPLICA for read replica." +
-            " cluster" +
-            ". " )
-    Setting<String> mode = setting( "dbms.mode", STRING, "SINGLE" );
```

Comment:
It has been moved (and converted to an enum) to [ClusterSettings.java](https://github.com/neo4j/neo4j/blob/c8932e8f5e8b1eda8f905bb8b33cc8059a16121e/enterprise/cluster/src/main/java/org/neo4j/cluster/ClusterSettings.java)

reason being that the mode setting must be accessible by all the cluster database creators, as well as the server endpoint. The cluster module satisfied that requirement, though it is not the perfect place for this setting, I agree.